airbyte-scheduler/src/main/java/io/airbyte/scheduler/persistence/DefaultJobCreator.java

```diff
@@ -127,16 +131,33 @@ public long createSyncJob(SourceConnection source,
                             String sourceDockerImageName,
                             String destinationDockerImageName)
       throws IOException {
-    final UUID connectionId = standardSync.getConnectionId();
+    return createSyncJobInternal(
+        standardSync.getConnectionId(),
+        source.getConfiguration(),
+        destination.getConfiguration(),
+        AirbyteProtocolConverters.toConfiguredCatalog(standardSync.getSchema()),
+        sourceDockerImageName,
+        destinationDockerImageName);
+  }
 
+  private long createSyncJobInternal(
```

Comment:
Something is up with your editor. It is not the first time I see weird formatting.

Reply:
![Screen Shot 2020-12-08 at 8 22 48 AM](https://user-images.githubusercontent.com/9092207/101510287-98d05100-392e-11eb-84eb-1e57eeb13cbf.png)

Am I using the wrong style plugin? I think this is what is described in the docs. https://docs.airbyte.io/contributing-to-airbyte/code-style.

Reply:
this seems like the sort of thing that :format should be fixing, no?

Reply:
i think i have added autoformat to my ide such that this will no longer be a problem.